### ISSUE TITLE
media query on "print" does not work

### ISSUE THREAD

**Issue by misjob** [PolymerElements/iron-media-query#17]
A media query on "print" does not work because the element automatically adds parenthesis to the query.

```javascript
if (query[0] !== '(') {
    query = '(' + query + ')';
}
```

**Comment by notwaldorf** [comment]
We should add an extra attribute called `full-query` that will skip the bracketing. So the usage would be
```html
<iron-media-query full-query="screen and (min-width: 480px)"></iron-media-query>
```

**Comment by bicknellr** [comment]
@notwaldorf fixed here: #21

**Comment by bicknellr** [comment]
see #21
